lib/containers/bats.pm

```diff
@@ -192,6 +192,10 @@ sub bats_setup {
         record_info("Disabling SELinux");
         assert_script_run "sed -i 's/^SELINUX=.*/SELINUX=permissive/' /etc/selinux/config";
         assert_script_run "setenforce 0";
+    } else {
+        # Rebuild SELinux policies without the so-called "dontaudit" rules
+        # https://en.opensuse.org/Portal:SELinux/Troubleshooting
+        script_run "semodule -DB";
```

Comment:
Why not `assert_script_run`? :thinking:

Reply:
> Why not `assert_script_run`? 🤔

This will fail on non-SELinux systems.

Reply:
```suggestion
        script_run "semodule -DB || true"; # Too lazy to add a check for non-SELinux systems so we just swallow the error
```
I would then add a `|| true` to make it clear, that we accept that this might fail.

Would not change the code but it might make it better understandable for our future selves.

Reply:
you might rely on `version_utils::has_selinux`

Reply:
Fixed.